app/Models/Foundation/Summit/TimeDurationRestrictedEvent.php

```diff
@@ -87,6 +88,18 @@ private function _setDuration(?Summit $summit, int $duration_in_seconds, bool $s
             throw new ValidationException(sprintf('Duration should be greater than %s minutes.', self::MIN_EVENT_MINUTES));
         }
 
+        $type = $this->getType();
+        if ($type instanceof PresentationType) {
```

Comment:
but this is not having in consideration the fact that for admins this restriction has no meaning, its only meant for track chairs

Reply:
done